falco.yaml

```diff
@@ -114,6 +114,25 @@
 # disabling the automatic artifacts followed by falcoctl.
 
 
+#######################
+# Falco configs files #
+#######################
+
+# [Stable] `configs_files`
+#
+# Falco will load additional configs files specified here.
+# Their loading is assumed to be made *after* main config file has been processed,
+# exactly in the order they are specified.
+# Therefore, loaded config files *can* override values from main config file.
+# Also, nested include is not allowed, ie: included config files won't be able to include other config files.
+# When relative path is used, the config file is assumed to be in the same folder as the main config file.
```

Comment:
I don't think it is the same behavior since for rules files there is no a "main" rule file concept.

Reply:
No check is done while reading rules files:
```
const std::string &filename = *it;
		std::ifstream is;
		is.open(filename);
		if (!is.is_open())
		{
			throw falco_exception("Could not open file " + filename + " for reading");
		}
```

Still, i think it makes much sense for configs files considering that here we actually have a `main` rule file concept.

Reply:
Also, `rules_file` throws an exception when a file cannot be found. Instead, `configs_files` just gives a warning. Shall we throw an exception for configs files too?

Reply:
1. I agree it must throw an exception when a config file is not found.
2. So, using a relative path in `rules_file` will make Falco use the current working dir, on the other hand, using a relative path in `configs_files` will make Falco use the main config path as the base path. Is my understanding correct?
  - TBH, I would have completely disallowed the ability to use relative path in such context. Anyway, I don't have strong opinions.
  - At least, please document the actual behavior for both cases :pray:

Reply:
> So, using a relative path in rules_file will make Falco use the current working dir, on the other hand, using a relative path in configs_files will make Falco use the main config path as the base path. Is my understanding correct?

Yep, otherwise we can also let configs files manage relative paths as just that, relative paths to cwd and we don't need to document anything because that's the expected behavior indeed :)

Reply:
Ok doing so 
> I agree it must throw an exception when a config file is not found.

Allows us to also drop the `warnings` parameter, which leads to a cleaner API :) 
I pushed a commit that:
* throws an exception when file is not found
* silently ignores non-regular files (and non directory)

Reply:
Updated PR body with final design: https://github.com/falcosecurity/falco/pull/3024#issue-2088412184